### ISSUE TITLE
Timeout time is too long to wait to get the exception

### ISSUE THREAD

**Issue by vinay95garg** [SeleniumHQ/selenium-ide#837]
## 🐛 Bug Report

Timeout time is too long to wait to get the exception

## To Reproduce

Recording anything
plays the recording 

change the screen, so that element should not be found and check the recorder, it will take an infinite time to give any exception.

I have attached one screenshot where I recorded something and then trying to play it and as it started playing I changed the tab so it is searching for infinite that thing which should not be like this.

## Expected behavior
It while playing recording we change the tab then it will not get that element so it should give an exception for a timeout after 30 seconds or 1 minute and it is not taken care inside 
selenium-ide/packages/selenium-ide/src/neo/components/LogMessage/index.jsx

## Project file reproducing this issue (highly encouraged)

source file that can be changed : selenium-ide/packages/selenium-ide/src/neo/components/LogMessage/index.jsx

Issues without a reproduction project are likely to stall.

## Environment

chrome browser
![inifnite](https://user-images.githubusercontent.com/13987555/65854807-27386280-e37b-11e9-8f9f-dcc210782dc2.PNG)

Selenium IDE Version: 3.13

**Comment by tourdedave** [comment]
Please attach a SIDE project file which demonstrates the issue and provide the missing details from the issue template about your environment specifics (e.g., operating system, browser version, etc.)

**Comment by vinay95garg** [comment]
> Please attach a SIDE project file which demonstrates the issue and provide the missing details from the issue template about your environment specifics (e.g., operating system, browser version, etc.)

You can record any script. The only issue is if you are playing that recorded script and while playing you change the tab manually then in execution it will start searching for that element and it will not found it since we changed the tab and it should give timeout after few seconds but it goes infinite

os is windows 8 
version v1.13

**Comment by vinay95garg** [comment]
[Timeout.zip](https://github.com/SeleniumHQ/selenium-ide/files/4004436/Timeout.zip)
I have uploaded the side file when you click on play option and in between of the recording you close the browser then it will go and try to find element infinitely instead of a timeout after a few seconds.

**Comment by tusharatravalkar** [comment]
I am also facing same kind of issue in case if server is down and not able to connect to the server- I was trying to verify Element Present/waitForText commands with Selenium IDE 3.17. It should failed test after default timeout(i.e. 30sec) but it didn't failed and it run for infinite time. How to handle this issue?

**Comment by tusharatravalkar** [comment]
> Does this relate to "Wait for element present" and it just sits and spins even though the element is on the page? If I press pause and then un-pause, it will then continue on as if it found the element.

Yes, this issue related for "Wait for element present" and 'Verify element present' and spin even element is not present after 30 seconds/specified time. This issue occurs only if server is down.  It should failed the test case.

**Comment by smith-test** [comment]
> > Does this relate to "Wait for element present" and it just sits and spins even though the element is on the page? If I press pause and then un-pause, it will then continue on as if it found the element.
> 
> Yes, this issue related for "Wait for element present" and 'Verify element present' and spin even element is not present after 30 seconds/specified time. This issue occurs only if server is down. It should failed the test case.

Thank you but I am wondering why all of a sudden Wait for element present stopped working in the past 10-14 days.  I noticed this thread's original issue was created last September.  Being new to this forum and Open Source software, what is the expected wait time for things to get fixed?  I understand it is open source, so it will depend on volunteers but I am wondering what is the norm.
